### ISSUE TITLE
TestImportMicrographs

### ISSUE THREAD

**Issue by yaizar** [I2PC/scipion#1860]
**Describe the bug**
[This line](https://github.com/I2PC/scipion/blob/devel-pluginization/pyworkflow/tests/em/protocols/test_protocols_import_micrographs.py#L154) checks if the attributes of two mic sets are equal, they're not because of the `voltage`:

```
>>>>> python  /home/yaiza/git/scipion/pyworkflow/apps/pw_protocol_run.py "/home/yaiza/ScipionUserData/projects/TestImportMicrographs" "Runs/000136_ProtImportMicrographs/logs/run.db" "136"
Different attributes: 
self._voltage = 200.0
other._voltage = 300.0
Different attributes: 
self._acquisition = 
    mag=10000.0
    volt= 200.0
    Cs=2.0
    Q0=0.1


other._acquisition = 
    mag=10000.0
    volt= 300.0
    Cs=2.7
    Q0=0.1


[   FAILED ] TestImportMicrographs.test_fromXmipp

Traceback (most recent call last):
  File "/home/yaiza/git/scipion/software/lib/python2.7/unittest/case.py", line 329, in run
    testMethod()
  File "/home/yaiza/git/scipion/pyworkflow/tests/em/protocols/test_protocols_import_micrographs.py", line 155, in test_fromXmipp
    goldFn=micsMd.replace('.xmd', '_gold.sqlite'))
  File "/home/yaiza/git/scipion/pyworkflow/tests/em/protocols/test_protocols_import_micrographs.py", line 56, in checkMicSet
    self.assertTrue(mic1.equalAttributes(mic2, verbose=True))
  File "/home/yaiza/git/scipion/software/lib/python2.7/unittest/case.py", line 422, in assertTrue
    raise self.failureException(msg)
AssertionError: False is not true
```

Can we just set the voltage of the import protocol to 200 to match the comparing set?

**Comment by azazellochg** [comment]
Default voltage and Cs should be 300 and 2.7 for most cryo data sets

**Comment by yaizar** [comment]
looks like this dataset has voltage 200 and we were advised to change the test, not the dataset. fixed in 4e82ec2
